Sito/gestione_profilo_utente.php

```diff
@@ -0,0 +1,300 @@
+<?php
+	require_once("php/dbaccess.php");
+    session_start();
+
+	if( isset($_SESSION['username'])) {
+	         $db = null;
```

Comment:
Quando c'è un errrore i campi vengono messi come placeholder e quindi tocca rimetterli tutti da capo si vedono solo in sottoimpressione. Bisogna invece che sia possibile modiificarli e quindi sono da mettere come value. Idem per il metodo di pagamento.
![image](https://user-images.githubusercontent.com/56229661/71554673-c0325480-2a22-11ea-8259-228e65a92d62.png)

Reply:
Ora i campi sono value solo quando c'è un errore che va corretto, in caso contrario sono dei placeholder, per facilitare l'inserimento senza cancellare nulla